src/App.js

```diff
@@ -1,13 +1,57 @@
 import React, { useState } from 'react';
+import {Switch, Route, NavLink} from 'react-router-dom';
 import './App.css';
+import Homepage from "./Pages/Homepage";
+import LoginPage from "./Pages/LoginPage";
+import Overview from "./Pages/Overview";
+import Blogpost from "./Pages/Blogpost";
+import {useHistory} from "react-router-dom";
 
 function App() {
   // We houden in de state bij of iemand is "ingelogd" (simpele versie)
   const [isAuthenticated, toggleIsAuthenticated ] = useState(false);
 
+    const history = useHistory()
+
+    function signOut() {
+        toggleIsAuthenticated(false);
+        history.push("/")
+    }
+
   return (
     <div>
       Maak hier jouw prachtige blog-applicatie!
+        <nav>
+            <ul>
+                <li><NavLink to="/">Homepage</NavLink></li>
+                <li><NavLink to="/login">Login page</NavLink></li>
+                {isAuthenticated ?
+                    <>
+                        <li><NavLink to="/blogposts">Overview</NavLink></li>
+                        <button type="button" onClick={signOut}>Uitloggen</button>
+                    </>
+                    : ""}
+                {/*<li><NavLink to="/blogpost/:blogId">Blogpost <= werkt NIET</NavLink></li>*/}
+            </ul>
```

Comment:
Hier had je volgens de opdracht de login pagina weg moeten halen als de gebruiker ingelogt is dit kan je mooi doen met de turnery operator aan de ene kant van de : wat er gebeurd als de gebruiker is ingelogt nl overzicht en uitloggen en aan de andere kant van de : wat er gebeurd als de gebruiker niet is ingelogt nl een login knop (waar jij nu de "" hebt staan)